library/koji_tag_inheritance.py

```diff
@@ -211,21 +224,23 @@ def remove_tag_inheritance(session, child_tag, parent_tag, priority,
     """
     result = {'changed': False, 'stdout_lines': []}
     current_inheritance = session.getInheritanceData(child_tag)
-    found_rule = {}
+    found_rules = []
     for rule in current_inheritance:
         if rule['name'] == parent_tag and rule['priority'] == priority:
-            found_rule = rule.copy()
+            found_rules.append(rule.copy())
             # Mark this rule for deletion
-            found_rule['delete link'] = True
-            result['stdout_lines'].append(
-                    'remove parent %s (%d)'
-                    % (found_rule['name'], found_rule['priority']))
-    if not found_rule:
+            found_rules[-1]['delete link'] = True
+    if not found_rules:
         return result
+    if len(found_rules) > 1:
+        raise ValueError('multiple inheritance rules matched! how?!')
+    result['stdout_lines'].append('remove inheritance link:')
```

Comment:
to be clear, I think this use of `describe_inheritance_rule()` is great. I'm just not sure about the `found_rule` -> `found_rules` thing above.